### PR TITLE
Release: 1.26.0

### PR DESCRIPTION
**Pull request by sethmlarson**
Please verify that the changelog entries look good. Specifically was a little unsure how to phrase some of the SecureTransport changes @hodbn.

@nateprewitt would be nice to get some testing from Requests before we release this if possible.

### CODE REVIEW

**Comment by nateprewitt** [comment]
@sethmlarson :+1: I've got some time tomorrow set aside to kick the tires.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,45 @@
 Changes
 =======
 
+1.26.0 (2020-11-??)
+-------------------
+
+* Added support for HTTPS proxies contacting HTTPS servers (PR #1923, PR #1806)
```
Comment (shazow) [inline CHANGES.rst:7]:
```suggestion
* Added support for HTTPS proxies contacting HTTPS servers (Pull #1923, Pull #1806)
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+
+* Deprecated negotiating TLSv1 and TLSv1.1 by default. Users that
+  still wish to use TLS earlier than 1.2 without a deprecation warning
+  should opt-in explicitly by setting ``ssl_version=ssl.PROTOCOL_TLSv1_1`` (Pull #2002)
+  **Starting in urllib3 v2.0 connections that receive a ``DeprecationWarning`` will begin to fail**
```
Comment (shazow) [inline CHANGES.rst:12]:
```suggestion
  **Starting in urllib3 v2.0: Connections that encounter a ``DeprecationWarning`` will fail**
```

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,45 @@
...
+
+* Deprecated ``Retry`` options ``Retry.DEFAULT_METHOD_WHITELIST``, ``Retry.DEFAULT_REDIRECT_HEADERS_BLACKLIST``
+  and ``Retry(method_whitelist=...)`` in favor of ``Retry.DEFAULT_ALLOWED_METHODS``,
+  ``Retry.DEFAULT_REMOVE_HEADERS_ON_REDIRECT``, and ``Retry(allowed_methods=...)``
+  (PR #2000) **Starting in urllib3 v2.0 the deprecated options will be removed**
```
Comment (shazow) [inline CHANGES.rst:17]:
```suggestion
  (Pull #2000) **Starting in urllib3 v2.0: Deprecated options will be removed**
```

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,45 @@
...
+
+* Added default ``User-Agent`` header to every request (PR #1750)
```
Comment (shazow) [inline CHANGES.rst:19]:
```suggestion
* Added default ``User-Agent`` header to every request (Pull #1750)
```

File: CHANGES.rst [hunk 5]
```diff
@@ -1,6 +1,45 @@
...
+
+* Added ``urllib3.util.SKIP_HEADER`` for skipping ``User-Agent``, ``Accept-Encoding``, 
+  and ``Host`` headers from being automatically emitted with requests (Pull #2018)
+
+* Collapse ``transfer-encoding: chunked`` request data and framing into
+  the same ``socket.send()`` call (PR #1906)
```
Comment (shazow) [inline CHANGES.rst:25]:
```suggestion
  the same ``socket.send()`` call (Pull #1906)
```

File: CHANGES.rst [hunk 6]
```diff
@@ -1,6 +1,45 @@
...
+
+* Send ``http/1.1`` ALPN identifier with every TLS handshake by default (PR #1894)
```
Comment (shazow) [inline CHANGES.rst:27]:
```suggestion
* Send ``http/1.1`` ALPN identifier with every TLS handshake by default (Pull #1894)
```

File: CHANGES.rst [hunk 7]
```diff
@@ -1,6 +1,45 @@
...
+
+* Properly terminate SecureTransport connections when CA verification fails (Pull #1977)
+
+* Don't emit an ``SNIMissingWarning`` when passing ``server_hostname=None``
+  to SecureTransport (Pull #1903)
+
+* Disabled requesting TLSv1.2 session tickets as they weren't being used by urllib3 (PR #1970)
```
Comment (shazow) [inline CHANGES.rst:34]:
```suggestion
* Disabled requesting TLSv1.2 session tickets as they weren't being used by urllib3 (Pull #1970)
```

File: CHANGES.rst [hunk 8]
```diff
@@ -1,6 +1,45 @@
...
+
+* Suppress ``BrokenPipeError`` when writing request body after the server
+  has closed the socket (PR #1524)
```
Comment (shazow) [inline CHANGES.rst:37]:
```suggestion
  has closed the socket (Pull #1524)
```

File: CHANGES.rst [hunk 9]
```diff
@@ -1,6 +1,45 @@
...
+
+* Wrap ``ssl.SSLError`` that can be raised from reading a socket (e.g. "bad MAC")
+  into an ``urllib3.exceptions.SSLError`` (PR #1939)
```
Comment (shazow) [inline CHANGES.rst:40]:
```suggestion
  into an ``urllib3.exceptions.SSLError`` (Pull #1939)
```

**Review by shazow — COMMENTED** [review]
Nothing of particular concern, but suggested changing the PR annotation to be consistent throughout the changelog, in case some poor soul relies on this phrasing for parsing somewhere. :)

Also minor clarity improvements.

**Comment by sethmlarson** [comment]
@shazow Good recommendations, thanks!

**Comment by sethmlarson** [comment]
Need to add references to Python 2.7 (and 3.5) are deprecated and will be removed in v2

**Review by pquentin — DISMISSED** [review]
Sounds good to me!

+1 on PR -> Pull

**Comment by sethmlarson** [comment]
@shazow @nateprewitt Fixed the PR -> Pull and added a note on 1.26.x being the last release stream supporting Python 2.

**Review by shazow — DISMISSED** [review]

**Review by pquentin — DISMISSED** [review]

**Review by nateprewitt — APPROVED** [review]
Requests looks good 🚀

**Comment by hodbn** [comment]
Looks great! 🚀 :shipit:

**Review by hodbn — APPROVED** [review]

**Comment by sethmlarson** [comment]
Looks like we're releasing today! :rocket: :shipit:

**Pull request merged**